widget/migrations/0004_auto_20151125_1011.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+
+
+class Migration(migrations.Migration):
```

Comment:
@waseem-omar do you know why this migration is included in this branch even though there's no model changes?